setup.ts

```diff
@@ -161,45 +165,44 @@ class Chain {
     async democracyPropose(
         account: AddressOrPair,
         func: (
-            registrarAccount: string | AccountId | Uint8Array
+            args: any
         ) => Promise<SubmittableExtrinsic<'promise', ISubmittableResult>>,
-        args: string | AccountId | Uint8Array,
+        args: any,
         value = DEFAULT_DEMOCRACY_PROPOSAL_FEE
     ) {
-        await this.connect();
-        // const toAddRegistrar = await self.identityAddRegistrar(registrarAccount);
-        const result = await func(args);
+        await self.connect();
+        // @ts-ignore
+        const result = await func(...args);
```

Comment:
We need to unpack function parameters. 
`func` is a function pointer, and `args` is an array of function parameters.
Suppose the declaration of `func` is `func(a: Type1, b: Type2)`. 
if we invoke `func` by calling `func(args)`. It implies the declaration of `func(Array arr)` , and `args = [a: Type1, b: Type2]`. The method will perform in a totally different way.


That's why we need to unpack the array